applications/RANSApplication/custom_elements/convection_diffusion_reaction_element.h

```diff
@@ -0,0 +1,727 @@
+//    |  /           |
```

Comment:
No cpp?

Reply:
I kept it like that because, I dont know the template arguments beforehand.

Reply:
But can't the implementation of the methods be implemented in a cpp and then do the explicit instantiation below?

Reply:
done :)